### DISCUSSION TITLE
Linux support is here!

### DISCUSSION THREAD

**Discussion by abose** [brackets-cont/brackets#113] [Announcements]
## Update: Linux support is here
get it from https://phcode.io

## Previous issue
Brackets support for Linux is coming soon. We are with the Linux community on this and are working on top priority for a Linux port. 

Adobe had ditched support for Linux in its final build of brackets due to a CEF platform limitation with the older CEF builds that Adobe used for brackets.
The Brackets community is now working on a larger platform migration to move away from CEF-based brackets-shell for Linux support. We expect to complete it by March 2022 and will have Linux builds for most Linux flavors soon after.
More details can be found here: https://github.com/brackets-cont/brackets/issues/26

follow  https://github.com/brackets-cont/brackets/projects/3 for electron migration status

**Comment by jhcarlos8888** [comment]
Espectacular!!!!! Thanks Sr

**Comment by Mesum-Hussain** [comment]
I think shipping it as Flatpaks or Snaps would be a good idea. That way it will work on almost all of the Linux distros.

**Comment by NotAlexanderrrr** [comment]
Hey, Abose might you have a specific date when the Linux support is coming I would help if there were a specific date for it.


@abose

**Reply by NotAlexanderrrr** [reply]
Wait, nvm i see you said you expect it to be done by March 2022

**Reply by NotAlexanderrrr** [reply]
Also, @abose do you have an email?

**Reply by abose** [reply]
Hi @NotAlexanderrrr Any help is very much welcome. I hang out more in the brackets discord group: https://discord.com/invite/rBpTBPttca , just drop a hi there if you need to get in touch any time.

Update: it looks like we will only be starting work on the new platform by march as most of our dev time is spent on putting back the backend infrastructure for brackets that adbe dropped.

**Reply by abose** [reply]
We are also working on a browser-based version of brackets that needs no installation, but it is still very much in alpha.

https://github.com/aicore/phoenix

**Comment by NotAlexanderrrr** [comment]
Could I maybe get a link to the browser version? Or by alpha, you mean not done yet?
________________________________

**Reply by abose** [reply]
Alpha is live in the link below, extension manager, live preview, and code hints for js are still wip, but have beta features like markdown preview. Do mention if you have any suggestions.

http://phoenix.core.ai/src

**Reply by Mesum-Hussain** [reply]
Good work! looks awesome. I am excited for when it comes to Linux and releases on the web.

**Comment by Fr-Dae** [comment]
What are your definition of "soon' please ?

**Comment by NotAlexanderrrr** [comment]
is this to me?
________________________________

**Reply by Fr-Dae** [reply]
for @abose [abos](https://github.com/abose)

**Comment by NotAlexanderrrr** [comment]
Also will there be a chrome os version for brackets
________________________________

**Reply by abose** [reply]
Phoenix will support chromeos too

**Comment by NotAlexanderrrr** [comment]
In march?
________________________________

**Reply by abose** [reply]
We are aiming for the March-may timeframe.

**Comment by mbowzeylo** [comment]
Please...Linux version!

**Comment by char4lie** [comment]
> Brackets support for Linux is coming soon. We are with the Linux community on this and are working on top priority for a Linux port. 
> 
> Adobe had ditched support for Linux in its final build of brackets due to a CEF platform limitation with the older CEF builds that Adobe used for brackets.
> The Brackets community is now working on a larger platform migration to move away from CEF-based brackets-shell for Linux support. We expect to complete it by March 2022 and will have Linux builds for most Linux flavors soon after.
> More details can be found here: https://github.com/brackets-cont/brackets/issues/26
> 
> follow  https://github.com/brackets-cont/brackets/projects/3 for electron migration status

That means termux users will be able to access bracket 
fuctions

**Comment by abose** [comment]
## Update Linux support is here

get it from https://phcode.io/
